packages/api/amplify_api/lib/src/graphql/web_socket/blocs/web_socket_bloc.dart

```diff
@@ -55,6 +55,10 @@ class WebSocketBloc with AWSDebuggable, AmplifyLoggerMixin {
   @override
   String get runtimeTypeName => 'WebSocketBloc';
 
+  /// Indicates if the bloc has finished closing
+  @visibleForTesting
+  final closed = Completer();
```

Comment:
Can we call it `done` to match `Sink.done` and mark it `<void>`

```suggestion
  final done = Completer<void>();
```

Instead of awaiting it in the test, can we await it in `ApiPlugin.reset`? And then in the test, we can await `Amplify.reset` to get the expected behavior.